File: 9-palindrome-number/9-palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        str_form = str(x)
        flag = True
        for i in range(len(str_form)):
            if str_form[i] != str_form[len(str_form)-1-i]:
                flag = False
        if flag == True:
            return True
        else:
            return False

File: 9-palindrome-number/9-palindrome-number.cpp
#include <iostream>  
#include <string>  

class Solution {
public:
    bool isPalindrome(int x) {
        string s1 = to_string(x);
        string s2 = to_string(x);
        reverse(s2.begin(),s2.end());
        return s1 == s2;
    }
};